### ISSUE TITLE
Access public_ports from ServiceConfig

### ISSUE THREAD

**Issue by nidhi-singh02** [kurtosis-tech/kurtosis#2497]
### Background & motivation

AFAIK currently ports are being exposed randomly if no `public_ports` is specified. Since these ports are random, do we have a way to access it from the service. 

I see a API method https://docs.kurtosis.com/engine-apic-reference/#getpublicports---mapportid-portspec which is possible in golang and Typescipt, I want to get the public port in my starlark package. 

### Desired behaviour

Access public ports of a service in starlark package

### How important is this to you?

Painful; the lack of this feature makes using Kurtosis frictionful.

### What area of the product does this pertain to?

Other: anything not covered by the above

**Comment by garyng2000** [comment]
yes, this makes it only good for local personal dev

**Comment by nidhi-singh02** [comment]
Yes, A feature request for this would be super helpful , when are you guys planning to have this one on the roadmap ?

**Comment by hlongvu** [comment]
yep, when add a FrontEnd service into the stack we often need to get the public ports of other services to make request to.
